registry/remote/errop/errors.go

```diff
@@ -0,0 +1,20 @@
+/*
+Copyright The ORAS Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package errop
+
+const (
+	DeleteDanglingReferrerIndex = "delete dangling referrers index"
```

Comment:
How are we planning to define error codes? 
Upper case underscore or something else. 
These constants needs to exposed is my understanding.

Reply:
> These constants needs to exposed is my understanding.

The value of this constants will not be exposed to end user. E.g. `"delete dangling referrers index"` here won't be visible to user, only `Err` might be visible in client output.